packages/ember-views/lib/views/select.js

```diff
@@ -50,7 +50,7 @@ var SelectOption = View.extend({
     } else {
       // Primitives get passed through bindings as objects... since
       // `new Number(4) !== 4`, we use `==` below
-      return value === get(this, 'attrs.parentValue');
+      return value == get(this, 'attrs.parentValue');
```

Comment:
The tests are failing JSHint (because we generally require `===` instead of `==`).  Can you add `// jshint ignore:line` to the end of the line so that those tests pass?